src/index.js

```diff
@@ -1,33 +1,66 @@
 import './style.css';
 
-const todo = [
-  {
-    description: 'Learn Javascript',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Complete JavaScript Projects',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Go to Gym',
+const getDataFromLocalStorage = () => {
+  const todoList = localStorage.getItem('todoList');
+  return JSON.parse(todoList);
+};
+
+const updateView = () => {
+  const todoList = getDataFromLocalStorage();
+  const todo = document.getElementById('todo-list');
+  todo.innerHTML = '';
+  if (todoList) {
+    todoList.forEach((todoListItem, index) => {
+      todo.innerHTML += `
+        <li><hr></li>
+        <li class="todo">
+            <div>
+                <input type="checkbox" name="checkbox" value="${index}" ${todoListItem.completed ? 'checked' : ''}>
+              <h2>${todoListItem.description}</h2>
```

Comment:
- [ ] **Required:** Your list items should have an index key. Kindly add it.

![image](https://user-images.githubusercontent.com/27709832/154589040-42b472be-6dde-44fc-94b6-ef2ceef7d7ee.png)
 
- [ ] **Required:** Kindly implement a function for editing the list item description.

- [ ] **Required:** [Kindly see the requirement for this milestone](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md) again for a refresher of all functionality that should be implemented.
This includes:

- ability to delete list items
- deleting an object should update the indexes of all other items.
- all changes to the To-Do List should be saved in local storage.

- [ ] **Optional:** Add an image/demo link to your project README